website/src/pages/docs/guides/mesh-sdk.mdx

```diff
@@ -9,21 +9,21 @@ You can use GraphQL Mesh as a completely type-safe SDK in your existing TypeScri
 Mesh's execute function could be used to directly carry out a query:
 
 ```ts
-import { getBuiltMesh } from './.mesh'
-
-async function executeQuery(document, variables) {
-  const { execute } = await getBuiltMesh();
-
-  return await execute(/* GraphQL */`
-    query myQuery($someVar: String!) {
-       foo(someArg: $someVar) {
-          bar
-       }
-    }
-  `, {
-     someArg: 'SOME_VALUE'
-  });
-}
+import { execute } from './.mesh'
+
+const exampleDocument = /* GraphQL */`
+  query myQuery($someVar: String!) {
+      foo(someArg: $someVar) {
+        bar
+      }
+  }
+`;
+
+const exampleVariables = {
+  someArg: 'SOME_VALUE'
+};
+
+execute(ducument, variables);
```

Comment:
```suggestion
const result = await execute(document, variables);
```
I think it is worth to mention `await` here. Since it has `await`, it is also worth to wrap this in an async `main` function then call it after with `catch`.